src/theme/DocVersionBanner/index.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import clsx from 'clsx';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Link from '@docusaurus/Link';
+import Translate from '@docusaurus/Translate';
+import {
+  useActivePlugin,
+  useDocVersionSuggestions,
+} from '@docusaurus/plugin-content-docs/client';
+import {ThemeClassNames} from '@docusaurus/theme-common';
+import {
+  useDocsPreferredVersion,
+  useDocsVersion,
+} from '@docusaurus/theme-common/internal';
+function UnreleasedVersionLabel({siteTitle, versionMetadata}) {
+  return (
+    <Translate
+      id="theme.docs.versions.unreleasedVersionLabel"
+      description="The label used to tell the user that he's browsing an unreleased doc version"
```

Comment:
```suggestion
      description="The label indicating an unreleased docs version"
```
There are also female/other developers, and we want to be inclusive in our writing. Avoiding the generic masculine is a good style that we want to follow.

Reply:
Sorry for overseeing this. This is a copy of the Docusaurus layout, so I haven't written it (it also won't be seen). 
I'll fix it though.
(same applies for majority of your review comments)